disk-utils/cfdisk.c

```diff
@@ -986,14 +986,15 @@ static size_t menuitem_get_line(struct cfdisk *cf, size_t idx)
 		if (!m->page_sz)				/* small menu */
 			return (ui_lines - (cf->menu->nitems + 1)) / 2 + idx;
 		return (idx % m->page_sz) + 1;
-	} else {
-		size_t len = MENU_H_ITEMWIDTH(m) + MENU_H_BETWEEN; /** item width */
-		size_t items = ui_cols / len;			/* items per line */
-
-		if (items == 0)
-			return 0;
-		return MENU_START_LINE + ((idx / items));
 	}
+
+	size_t len = MENU_H_ITEMWIDTH(m) + MENU_H_BETWEEN; /** item width */
+	size_t items = ui_cols / len;			/* items per line */
+
```

Comment:
This against our coding style, we define variables at beginning of the code block and this is the reason why there is else {}. Please, keep else {} there in this case.